src/tcgwars/logic/impl/gen2/AquapolisNG.groovy

```diff
@@ -0,0 +1,2961 @@
+  LUGIA_149 ("Lugia", "149", Rarity.RARE, [POKEMON, BASIC, _COLORLESS_]),
+  NIDOKING_150 ("Nidoking", "150", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _COLORLESS_]),
+  AMPHAROS_H1 ("Ampharos", "H1", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _LIGHTNING_]),
+  EXEGGUTOR_H10 ("Exeggutor", "H10", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _GRASS_]),
+  HOUNDOOM_H11 ("Houndoom", "H11", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _DARKNESS_]),
+  HYPNO_H12 ("Hypno", "H12", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _PSYCHIC_]),
+  JUMPLUFF_H13 ("Jumpluff", "H13", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _GRASS_]),
+  KINGDRA_H14 ("Kingdra", "H14", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _WATER_]),
+  LANTURN_H15 ("Lanturn", "H15", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _LIGHTNING_]),
+  MAGNETON_H16 ("Magneton", "H16", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _METAL_]),
+  MUK_H17 ("Muk", "H17", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _GRASS_]),
+  NIDOKING_H18 ("Nidoking", "H18", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _FIGHTING_]),
+  NINETALES_H19 ("Ninetales", "H19", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _FIRE_]),
+  ARCANINE_H2 ("Arcanine", "H2", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _FIRE_]),
+  OCTILLERY_H20 ("Octillery", "H20", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _WATER_]),
+  SCIZOR_H21 ("Scizor", "H21", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _METAL_]),
+  SLOWKING_H22 ("Slowking", "H22", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _PSYCHIC_]),
+  STEELIX_H23 ("Steelix", "H23", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _METAL_]),
+  SUDOWOODO_H24 ("Sudowoodo", "H24", Rarity.RARE, [POKEMON, BASIC, _FIGHTING_]),
+  SUICUNE_H25 ("Suicune", "H25", Rarity.RARE, [POKEMON, BASIC, _WATER_]),
+  TENTACRUEL_H26 ("Tentacruel", "H26", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _WATER_]),
+  TOGETIC_H27 ("Togetic", "H27", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _COLORLESS_]),
+  TYRANITAR_H28 ("Tyranitar", "H28", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _DARKNESS_]),
+  UMBREON_H29 ("Umbreon", "H29", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _DARKNESS_]),
+  ARIADOS_H3 ("Ariados", "H3", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _GRASS_]),
+  VICTREEBEL_H30 ("Victreebel", "H30", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _GRASS_]),
+  VILEPLUME_H31 ("Vileplume", "H31", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _GRASS_]),
+  ZAPDOS_H32 ("Zapdos", "H32", Rarity.RARE, [POKEMON, BASIC, _LIGHTNING_]),
+  AZUMARILL_H4 ("Azumarill", "H4", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _WATER_]),
+  BELLOSSOM_H5 ("Bellossom", "H5", Rarity.RARE, [POKEMON, EVOLUTION, STAGE2, _GRASS_]),
+  BLISSEY_H6 ("Blissey", "H6", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _COLORLESS_]),
+  ELECTRODE_H7 ("Electrode", "H7", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _LIGHTNING_]),
+  ENTEI_H8 ("Entei", "H8", Rarity.RARE, [POKEMON, BASIC, _FIRE_]),
+  ESPEON_H9 ("Espeon", "H9", Rarity.RARE, [POKEMON, EVOLUTION, STAGE1, _PSYCHIC_]);
```

Comment:
Really minor detail, but could these be reordered? here and in the switch cases below for the 3 e-Card sets.

Reply:
I've reordered the H_ cards

Reply:
Seems Expedition has no H_ cards, but Skyridge does. Aquapolis looks 👍